charts/workflows/charts/move2kube/templates/01-sonataflow_m2k.yaml

```diff
@@ -2,15 +2,15 @@ apiVersion: sonataflow.org/v1alpha08
 kind: SonataFlow
 metadata:
   annotations:
-    sonataflow.org/description: Workflow to execute Move2Kube
+    sonataflow.org/description: Workflow to execute {{ .Values.workflow.name }}
     sonataflow.org/expressionLang: jq
     sonataflow.org/version: "1.0"
   creationTimestamp: null
   labels:
-    app: m2k
-    sonataflow.org/workflow-app: m2k
-  name: m2k
-  namespace: default
+    app: {{ .Values.workflow.name }}
+    sonataflow.org/workflow-app: {{ .Values.workflow.name }}
+  name: {{ .Values.workflow.name }}
+  namespace: {{ .Values.namespace }}
```

Comment:
why would we need to paramerize the name of a well known flow that fits a specific image?
aside the fact that this will alway conflict and will be overridden by the manifest generation this is almost never what we want.
what it the use case you have in mind?

Reply:
Some naming (in this case, the `.Values.workflow.name`) in others places so I do need to put them as values
`.Values.namespace` is also used to make sure all resources are deployed in the same namespace, not sure it's needed though (my main concern was SA perms across namespace)

Reply:
hum, namespace should not be in the chart actually (https://github.com/helm/helm/issues/5465) I'll remove

Reply:
namespace can be set by the helm installation. do you need to override that?

Reply:
?
I removed all notion of namespace and  added the `--namespace` option in the install.md